README.md

```diff
@@ -46,3 +46,28 @@ Note that the `DynamoDB` operations are _upsert_, so if an issue happens, the la
 ## Issues
 
 Please report issues in the [elastic-ipfs/elastic-ipfs repo](https://github.com/elastic-ipfs/elastic-ipfs/issues).
+
+## Contribute and release process
+
+This is the process to contribute and get a new build deployed:
+
+- Create a Pull Request with the intended change
```

Comment:
```suggestion
- Create a Pull Request with the intended change
  - Update `package.json` and `package-lock.json` version
```

Reply:
I did not do this last PR, but from @francardoso93 example yesterday for bitswap peer, perhaps we should use same process here? thoughts @simone-sanfratello ?

Reply:
Lambdas are not versioned on the deploy, bitswap is - @francardoso93 correct me if I'm wrong